imsim/camera.py

```diff
@@ -160,14 +181,31 @@ class Camera(dict):
     Class to represent the LSST Camera as a dictionary of CCD objects,
     keyed by the CCD name in the focal plane, e.g., 'R01_S00'.
     """
-    def __init__(self, camera_class='LsstCam'):
+    _req_params = {'camera': str}
+    _opt_params = {'bias_levels_file': str}
+
+    def __init__(self, camera='LsstCam', bias_levels_file=None):
         """
         Initialize a Camera object from the lsst instrument class.
         """
         super().__init__()
-        self.lsst_camera = get_camera(camera_class)
+        self.lsst_camera = get_camera(camera)
+        if bias_levels_file is not None:
+            if not os.path.isfile(bias_levels_file):
+                bias_levels_file = os.path.join(data_dir, bias_levels_file)
+                if not os.path.isfile(bias_levels_file):
+                    raise FileNotFoundError(
+                        f"{bias_levels_file} not found.")
+            with open(bias_levels_file) as fobj:
+                bias_levels = json.load(fobj)
+        else:
+            # Set the input dict of per-amp bias levels to None.
+            bias_levels = defaultdict(lambda : None)
```

Comment:
There don't seem to be any tests for most of these new code paths.  I think all the unit tests still only test the else branch here, which doesn't use the bias_levels_file.  You should add something that tests the thing the main config file does.  And also one where the file name is fully specified (rather than implicitly in data_dir).  And ideally one that checks the OSError when it's not present.